internal/collector/processor/tailsampling/processor.go

```diff
@@ -189,7 +192,7 @@ func (tsp *tailSamplingSpanProcessor) ProcessSpans(td data.TraceData, spanFormat
 	idToSpans := make(map[traceKey][]*tracepb.Span)
 	for _, span := range td.Spans {
 		if len(span.TraceId) != 16 {
-			tsp.logger.Warn("Span without valid TraceId", zap.String("spanFormat", spanFormat))
+			tsp.logger.Warn("Span without valid TraceId")
```

Comment:
That information should be on the ctx, it seems good information to be on the warning message.